security-jwt/src/main/java/io/micronaut/security/token/jwt/bearer/BearerTokenConfigurationProperties.java

```diff
@@ -38,7 +38,7 @@
 
     private boolean enabled = DEFAULT_ENABLED;
     private String headerName = HttpHeaders.AUTHORIZATION;
-    private String prefix = HttpHeaderValues.AUTHORIZATION_PREFIX_BEARER;
+    private String headerPrefix = HttpHeaderValues.AUTHORIZATION_PREFIX_BEARER;
```

Comment:
why rename this? Should not be better to keep it with the same name as the getters and setters?

Reply:
duplicate of https://github.com/micronaut-projects/micronaut-security/pull/943#discussion_r819199988

Reply:
It "cognitively" conflicts with the constant PREFIX. Since the constant is public and can't be renamed, I renamed the field.